.github/workflows/testing-and-deployment.yml

```diff
@@ -26,15 +36,31 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Build package
+      - name: Setup dependencies
+        run: |
+          pip install -r requirements_build.txt --only-binary="numpy" --find-links https://wheels.pyvista.org/
+
+      - name: Build wheel
         run: |
-          pip install -r requirements_build.txt
           python setup.py bdist_wheel
-          pip install dist/*
 
-      - name: Install package
+      - name: Manually install VTK wheel on 3.10 since it does not work locally
+        run: |
+          pip install --upgrade pip
+          curl https://wheels.pyvista.org/vtk-9.1.0.dev0-cp310-cp310-macosx_12_0_x86_64.whl -o vtk-9.1.0.dev0-cp310-cp310-macosx_10_16_x86_64.whl
+          pip install ./vtk-9.1.0.dev0-cp310-cp310-macosx_10_16_x86_64.whl
+        if: ${{ matrix.os == 'macos-12' }}
```

Comment:
Locally on my 12.3 Monterey install I cannot reproduce this issue, but on GH actions there is some problem installing this wheel despite many attempts at debugging. So here I just hack the filename to make it seem like at 10_16 wheel so that `pip` will install it.

It's a hack but I think it's acceptable since we want a 3.10 wheel for this package, and this hack can go away once VTK releases 9.2 with 3.10 wheels in the coming weeks (!?)